go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/andreyvit/diff v0.0.0-20170406064948-c7f18ee00883
 	github.com/apparentlymart/go-cidr v1.1.0
 	github.com/aws/aws-sdk-go v1.34.10
+	github.com/aws/aws-sdk-go-v2 v0.18.0
```

Comment:
`v2` is an experimental version of aws-sdk-go, and we shouldn't depend on both versions, prefer v1